guides/common/modules/proc_upgrading-smartproxy-server.adoc

```diff
@@ -139,3 +139,19 @@ include::snip_steps-needs-reboot.adoc[]
 ifdef::orcharhino,satellite[]
 . Optional: If you use custom repositories, ensure that you enable these custom repositories after the upgrade completes.
 endif::[]
+
+.Upgrading {SmartProxyServersTitle} using remote execution
+
+. Create a backup or take a snapshot.
++
+For more information on backups, see {AdministeringDocURL}backing-up-satellite-server-and-capsule-server[Backing Up {ProjectServer} and {SmartProxyServer}] in the _Administering {ProjectName} {ProjectVersionPrevious}_ guide.
+. In the {ProjectWebUI} , navigate to *Monitor* > *Jobs*.
+. Click *Run Job*.
+. From the *Job category* list, select *Maintenance Operations*.
+. From the *Job template* list, select *{SmartProxy} Upgrade Playbook*.
+. In the *Search Query* field, enter the host name of the {SmartProxy}.
+. Ensure that *1 host* is displayed in the *Resolves to* field.
+. In the *target_version* field, enter the target version of the {SmartProxy}.
+. In the *whitelist_options* field, enter the options.
+. For *Type of query*, click *Static Query*.
```

Comment:
This looks odd. We do not normally use "for" in this way ("For Type of query"). 

What is "Type of query"? A web page? Pane? Section? List?

What is "Static Query"? A button? Radio button? Checkbox? If it is an ordinary button, "click" is correct. If it is a something else, then "select" would be correct.

Reply:
For reference, it looks like this:
![image](https://github.com/theforeman/foreman-documentation/assets/108661422/8584ba93-668f-428e-9c53-e39327e38f77)

Reply:
Great! I recommend this, as long as it aligns with the rest of the procedure and other documentation: "In the **Query type** section, select **Static query**." Radio buttons are selected/cleared, not clicked.

I hope those label subtitles ("evaluates ...") are mockups. They need a bit of work.

Reply:
Sadly not, what you are looking at is the production environment of 6.14 Satellite, possibly other versions too. I do not recollect in which version the job wizard was changed.

Reply:
OK, I will look at the UI when I have time and create a bug.

Reply:
Created [issue](https://projects.theforeman.org/issues/37220) to fix this.